options.go

```diff
@@ -186,10 +166,15 @@ func WithHealthz() Option {
 			}
 			if len(errs) > 0 {
 				s.logger.Warn("Ready check failed", zap.Any("errs", errs))
-				c.JSON(http.StatusServiceUnavailable, gin.H{"errors": errs})
-				return
+				b, err := json.Marshal(map[string]interface{}{"errors": errs})
```

Comment:
`s.logger.Warn("Ready check failed", zap.Any("errs", errs))` → `s.logger.Warn("Ready check failed", zap.Errors("errs", errs))`?

Reply:
Yes, but that will be my fourth PR ;-)